fs/copy_test.ts

```diff
@@ -47,7 +47,7 @@ function testCopySync(name: string, cb: (tempDir: string) => void) {
 }
 
 testCopy(
-  "[fs] copy file if it does no exist",
+  "copy() throws if src does not exist",
```

Comment:
```suggestion
  "copy() rejects if src does not exist",
```
Ditto for all other async functions which reject.